code/python/main-workshop/cdk_workshop/cdk_workshop_stack.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
         super().__init__(scope, id, **kwargs)
 
         # Defines an AWS Lambda resource
-        my_lambda = _lambda.Function(
```

Comment:
In all the workshop guides this is named `hello` so for sake of consistency I updated this name.